WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -475,6 +475,20 @@ public Boolean get() {
         return isElementDisplayed(elementID);
     }
 
+    public static boolean waitForElementToBeDisplayedWithoutFailure(final ViewInteraction element) {
+        try {
+            waitForConditionToBeTrueWithoutFailure(new Supplier<Boolean>() {
+                @Override
+                public Boolean get() {
+                    return isElementDisplayed(element);
+                }
+            });
+        } catch (Exception e) {
+            // ignore the failure
+        }
+        return isElementDisplayed(element);
+    }
+
```

Comment:
I had to do [the same thing in my PR](WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java). The only difference was that I made `waitForElementToBeDisplayedWithoutFailure(final Integer elementID)` call `waitForElementToBeDisplayedWithoutFailure(final ViewInteraction element)` to avoid a bit of code duplication. 
Nothing to do here.